src/applications/ask-a-question/config/form.js

```diff
@@ -53,13 +55,21 @@ const submitTransform = (formConfig, form) => {
 };
 
 const formConfig = {
+  rootUrl: manifest.rootUrl,
   urlPrefix: '/',
   submitUrl: `${environment.API_URL}/v0/ask/asks`,
   transformForSubmit: submitTransform,
   trackingPrefix: 'complex-form-',
   introduction: IntroductionPage,
   confirmation: ConfirmationPage,
   formId: '0873',
+  saveInProgress: {
+    messages: {
+      // inProgress: 'Your [savedFormDescription] is in progress.',
+      // expired: 'Your saved [savedFormDescription] has expired. If you want to apply for [benefitType], please start a new [appType].',
+      // saved: 'Your [benefitType] [appType] has been saved.',
+    },
+  },
```

Comment:
Does leaving `messages` as an empty object inside `saveInProgress` mean anything different than commenting `messages` out?

Reply:
No, this was just me being inconsistent. :sweat_smile: It's functionally the same.